main.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+from pymongo import MongoClient
+
+app = Flask(__name__, static_url_path='/static')
+client = MongoClient(app.config['MONGO_URI'])
+db = client.get_default_database()
+
+
+@app.route('/')
+def login():
+    return render_template('login.html')
+
+
+@app.route('/index/', methods=['POST'])
+def index():
+    if request.method == 'POST':
+        username = request.form['username']
+        password = request.form['password']
+
+        user = db.users.find_one({'username': username})
```

Comment:
make a file db_handler with functions such as 
`def fetch_user_by_username(username: str) -> UserModel` you should use an orm such as Beanie to use the models properly..

Reply:
if you really sharp you will realize having db_handler.py will not be enough and each model should have its own 'dal file'

Reply:
done i hope i did it right